backend/src/seeds/transactionLink/index.ts

```diff
@@ -5,7 +5,7 @@ export const transactionLinks: TransactionLinkInterface[] = [
     email: 'bibi@bloxberg.de',
     amount: 19.99,
     memo: 'Leider wollte niemand meine Gradidos zum Neujahr haben :(',
-    createdAt: new Date(2022, 0, 1),
+    createdAt: new Date(2022, 1, 1),
```

Comment:
I allowed myself to fix this @ogerly @Mogge

Reply:
What fix? What is wrong about January?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date#syntax

Reply:
Please take very care about javascript months!